docs/administrators_guide.md

```diff
@@ -78,6 +78,28 @@ membershipLists:
 in the config. Users can choose to disable this on a per-room basis by modfiying their
 [room config](./room_configuration.md#allowunconnectedmatrixusers) options, if the bridge permits it.
 
+## Subscribing to Moderation policys
```

Comment:
```suggestion
## Subscribing to Moderation policies
```

Reply:
Or are both correct?

Reply:
I think Policies is more correct than Policys